playbooks/ci-full-rhel/templates/heat_stack.yml

```diff
@@ -0,0 +1,230 @@
+    default: {{ ansible_env.testenv_network | default('internal') }}
+  floating_ip_pool:
+    type: string
+    description: Name of floating ip pool to use
+    default: {{ ansible_env.testenv_floating_ip_pool | default('external') }}
+  key_name:
+    type: string
+    description: Name of the ssh keypair to use for servers
+    default: {{ ansible_env.testenv_instance_prefix }}_key
+  security_group:
+    type: string
+    description: Name of the security group to use for servers
+    default: {{ ansible_env.testenv_instance_prefix }}_ursula
+  controller-0_name:
+    type: string
+    description: Name of controller-0 server
+    default: {{ ansible_env.testenv_instance_prefix }}-controller-0
+  controller-1_name:
+    type: string
+    description: Name of controller-1 server
+    default: {{ ansible_env.testenv_instance_prefix }}-controller-1
+  compute-0_name:
+    type: string
+    description: Name of compute-0 server
+    default: {{ ansible_env.testenv_instance_prefix }}-compute-0
+
+resources:
+
+  server_security_group:
+    type: OS::Neutron::SecurityGroup
+    properties:
+      description: Security group for servers of Ursula Tests
+      name: { get_param: security_group }
+      rules:
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 22
+          port_range_max: 22
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 80
+          port_range_max: 80
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 443
+          port_range_max: 443
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 35357
+          port_range_max: 35357
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 5000
+          port_range_max: 5000
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 6080
+          port_range_max: 6080
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 8774
+          port_range_max: 8774
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 8776
+          port_range_max: 8776
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 8777
+          port_range_max: 8777
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 9393
+          port_range_max: 9393
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 9292
+          port_range_max: 9292
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: tcp
+          port_range_min: 9696
+          port_range_max: 9696
+        - remote_ip_prefix: 0.0.0.0/0
+          protocol: icmp
+        - ethertype: 'IPv4'
+          remote_mode: 'remote_group_id'
+        - ethertype: 'IPv6'
+          remote_mode: 'remote_group_id'
+
```

Comment:
Need to also add below. Check with Perry he had made a PR  https://github.com/blueboxgroup/ursula/pull/2452
        - ethertype: IPv4
          remote_ip_prefix: 192.168.0.0/22
          protocol: igmp